pages/common/hg.md

```diff
@@ -1,7 +1,7 @@
 # hg
 
 > A command line interface for Mercurial, a distributed source control management system.
-> See the `hg-add` and `hg-commit` pages for additional information.
+> See the `hg-clone`, `hg-add` and `hg-commit` pages for additional information.
```

Comment:
Maybe you should add `and other pages` because it does not make sense to keep on adding commands here as you add more pages. People should get the general idea that other pages exist and they can just tldr it.

Reply:
Makes sense. :+1: